### PR TITLE
CRAN submit of 1.1.1

### PR DESCRIPTION
**Pull request by roninsightrx**
- Added some info for CRAN reviewer
- Added NEWS
- updated suggestion from checks to update the way classes are checked (using `inherit()`) 
- removed the stripping of the compiled files, since that seems to be a violation of CRAN policy.

### CODE REVIEW

File: R/sim.R [hunk 1]
```diff
@@ -292,18 +292,18 @@ sim <- function (ode = NULL,
     size <- attr(ode, "size")
   }
   if (!is.null(omega)) {
-    if(class(omega) == "matrix") {
+    if(inherits(omega, "matrix")) {
       omega_mat <- omega
     }
-    if(class(omega) %in% c("numeric", "integer")) {
+    if(inherits(omega, "numeric")) {
```
Comment (jasmineirx) [inline R/sim.R:298]: it's very unlikely that omega is ever going to be an integer, but if you want to catch them both you can use `mode`:

```suggestion
    if(mode(omega) == "numeric") {
```

mode will always return a logical of length 1.

File: cran-comments.md [hunk 2]
```diff
@@ -32,6 +32,24 @@ Namespace in Imports field not imported from: 'BH'
 
 Maintainer response: The BH package provides Boost C++ library headers. The BH library is not used in any of the code in /R or /src, but only in the C++ templates that comes with the package (in /inst/cpp). The C++ templates are used when creating a model using the new_ode_model() function, which is then compiled on-the-fly. If the BH package is removed from the Depends field, it won't be installed and hence models cannot be compiled and run. BH cannot be moved to the LinkingTo field.
 
+* CRAN repository db overrides:
+  X-CRAN-Comment: Archived on 2022-01-19 for policy violation.
+
+  On not remving debugging information.
```
Comment (jasmineirx) [inline cran-comments.md:38]:
```suggestion
  On not removing debugging information.
```

**Review by jasmineirx — APPROVED** [review]
lgtm. switching to using `inherits` is good practice. the review comments are addressed.

i added a couple minor suggestions.

File: cran-comments.md (hunk 2 above)
Comment (roninsightrx) [reply cran-comments.md:38]: that typo was in the CRAN output, so have to keep...

**Comment by roninsightrx** [comment]
add link to Github in DESCRIPTION

**Pull request merged**